roles/dg-config/tasks/main.yml

```diff
@@ -0,0 +1,198 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- name: DG | SQL to add standby logs
+  set_fact:
+    sql_for_dg:  |
+      alter system set dg_broker_start = true;
+      alter system set standby_file_management = auto;
+      begin
+      for c in (
+        select bytes, thr, (max(cnt_log) - nvl(max(cnt_sby), 0)) diff from (
+        select bytes, thread# thr, count(*) cnt_log, null cnt_sby from v\$log where thread# = 1 group by bytes, thread#
```

Comment:
Just thread 1?

Reply:
in current implementation the role creates DG using SI primary, however I put column thread# thr there as well for future change but restricted it to thread # = 1 only.
However this particular SQL to standby logs won't damage if standby logs created for other threads if they exist.
I will change it